client/app/services/notification.js

```diff
@@ -0,0 +1,22 @@
+import notification from 'antd/lib/notification';
+
+notification.config({
+  placement: 'bottomRight',
+  duration: 3,
+});
+
+const updatedActions = [
+  'success',
+  'error',
+  'info',
+  'warning',
+  'warn',
+].reduce((acc, action) => ({
+  ...acc,
+  [action]: (message, description = null, props = null) => notification[action]({ ...props, message, description }),
+}), null);
```

Comment:
This is really clever, but I have to admit that I had to read this carefully and really think about what's going on here. I kind of knew what it should do, but still had to unpack all the logic in my head.

How about the following as a simplified solution?

```javascript
const simpleNotification = {};

['success', 'error', ...].forEach(action => {
  simpleNotification[action] = (message, description = null, props = null) => notification[action]({ ...props, message, description });
});

export default {...notification, ...simpleNotification};
```

Reply:
+1

Reply:
BTW - how about code completion? Does your IDE recognize this as object with methods?

Reply:
@gabrieldutra  So when you type `notification.` somewhere in code, and then press `Ctrl+Space` - it shows `success`, `error`, ... in dropdown? Did I understand you right? That's not something crucial, but usually nice to have.

Reply:
Sorry @kravets-levko, I actually didn't write all the stuff that were on my mind :sweat_smile:. But what I forgot to mention is that code completion (Either with `ctrl+space` or just pressing a dot after `notification`) works for VSCode.

However, I had another issue... it only recognized the code and showed suggestions when I used the Relative Path to import (`import notification from '../../services/notification'`). Not sure about the reason, but I may be missing some extension here.